torq.q

```diff
@@ -565,6 +562,18 @@ sys:{[cmd]
 	.lg.o[`system;"executing system command: ",cmd];
 	catcherror:{[cmd;error] .lg.e[`system;"failed to execute ",cmd,": ",error];'error};
 	@[{result:system x;.lg.o[`system;"successfully executed"];result};cmd;catcherror[cmd;]]
+	/-@[{result:system x;.lg.o[`system;"successfully executed"];result};cmd;{.lg.e[`system;"failed to execute ",cmd,": ",x];'x}]
+	};
+
+// Load additional config files, check if the specific configuration environment variable exists and load the settings if it does
+loadaddcnfg:{[envvar;cnfgpath]
+	evstring:string envvar;
+	$[""~getenv[envvar];
+	.lg.o[`fileload;"environment variable ",evstring," not set, not loading app specific config"];
```

Comment:
I think we can tidy things up a bit here if we use an `if` instead of ` $` and return early in this case e.g.

```suggestion
	if[""~getenv[envvar];
		.lg.o[`fileload;"environment variable ",evstring," not set, not loading app specific config"];
		:();
	];
```

then the rest of the code doesn't need to be wrapped in a block